sway/commands/seat.c

```diff
@@ -7,15 +7,45 @@
 #include "stringop.h"
 
 // must be in order for the bsearch
+// these handlers perform actions on the seat
+static struct cmd_handler seat_action_handlers[] = {
+	{ "cursor", seat_cmd_cursor },
+};
+
+// must be in order for the bsearch
+// these handlers alter the seat config
 static struct cmd_handler seat_handlers[] = {
 	{ "attach", seat_cmd_attach },
-	{ "cursor", seat_cmd_cursor },
```

Comment:
Adding a `bool` flag to `cmd_handler` would have been much nicer… But then it wouldn't work with `config_subcommand` and `find_handler`.

Reply:
Actually, `find_handler` has some issues that need to be fixed anyway. For example, `seat * layout tabbed` works the same as `layout tabbed` (plus the unnecessary seat overhead)